client/src/pages/private/ParticipantTable.js

```diff
@@ -190,6 +196,16 @@ const ParticipantTable = () => {
   const isAdmin = isMoH || isSuperUser;
   const isEmployer = roles.includes('health_authority') || roles.includes('employer');
 
+  const useStyles = makeStyles((theme) => ({
+    cellTextStrong: {
+      fontWeight: 'bold',
+    },
+    cellTextWeak: {
```

Comment:
nit: maybe `cellTextGray` sounds beeter?

Reply:
Much better! Thanks for the suggestion, I didn't like like what I had either.